src/_gettsim/functions_loader.py

```diff
@@ -431,31 +532,20 @@ def _create_one_aggregation_func(  # noqa: PLR0912
         Name of the aggregated column.
     agg_specs : dict
         Dictionary of aggregation specifications. Can contain the source column
-        ("source_col") and the group ids ("group_id")
+        ("source_col") and the group ids ("group_id") Dictionary of aggregation
+        specifications. Must contain the aggregation type ("aggr"). Unless
+        `aggr == "count"`, it must contain the column to aggregate ("source_col").
     user_and_internal_functions: dict
         Dictionary of functions.
 
 
     Returns
     -------
-    aggregation_func : The aggregation func with the expected signature
+    aggregate_by_group_func : The aggregation func with the expected signature
 
     """
-    # Read individual specification parameters and make sure nothing is missing
-    try:
-        aggr = agg_specs["aggr"]
-    except KeyError as e:
-        raise KeyError(
-            f"No aggr keyword is specified for aggregation column {agg_col}."
-        ) from e
 
-    if aggr != "count":
-        try:
-            source_col = agg_specs["source_col"]
-        except KeyError as e:
-            raise KeyError(
-                f"Source_col is not specified for aggregation column {agg_col}."
-            ) from e
+    _check_agg_specs_validity(agg_specs, agg_col)
```

Comment:
This should go in a comprehension one level further up, symmetric to `aggregate_by_p_id`.